### PR TITLE
Remove the need for banned.php

### PR DESCRIPTION
**Pull request by Apogate**
Move banned.php in to imgboard.php.

One less file to maintain

### CODE REVIEW

File: imgboard.php [hunk 1]
```diff
@@ -70,6 +70,12 @@ function error($mes, $dest, $fancy = 0) {
         $del = new SaguaroDelete;
         $del->userDel();
         break;
+    case 'banned':
+        require_once(CORE_DIR . "/admin/bans.php");
+        $ban  = new Banish;
```
Comment (RePod) [inline imgboard.php:75]: Questionable indenting. 

I'm not one for enforcing style as long as it's semi-consistent but right now I'm really not feeling this.

File: banned.php [hunk 2]
```diff
@@ -1,24 +0,0 @@
...
-$mysql->init();
-
-//Init bans
-require_once(CORE_DIR . "/admin/bans.php");
-$ban  = new Banish;
-
-//Init page class. repod whispers "finally" somehwere.
-require_once(CORE_DIR . "/page/page.php");
-$page = new Page;
-$page->headVars['page']['title'] = "You are not banned!";
```
Comment (RePod) [inline banned.php:17]: Is this just being skipped from now on?

What is it doing currently?

**Review by RePod — COMMENTED** [review]
Keep in mind it's never been about the "number of files" we need to maintain, but purely the amount of unoptimized code.

This change removes one file by tucking its effective code into `imgboard.php` directly but we'll still need to look at `/admin/bans.php` to know what it's actually doing.

File: imgboard.php (hunk 1 above)
Comment (Apogate) [reply imgboard.php:75]: this is how we've been indenting for the past 2 years

File: banned.php (hunk 2 above)
Comment (Apogate) [reply banned.php:17]: it's all done in /admin/bans.php in the banScreen function

**Comment by RePod** [comment]
If it works fine, then merge.

**Pull request merged**